### DISCUSSION TITLE
Request for not "optimizing" the html-span tag (or others)

### DISCUSSION THREAD

**Discussion by doctomlug** [freeplane/freeplane#1371] [Ideas]
Sorry for being late to the html tag party.

My scripts use <span> tags for highlighting etc. and it is important for the script to be able to detect its "own" highlighting (e.g. to remove it selectively). So far I used the class attribute as an indicator, as the span tag was not removed and the attributes were left as is this worked well. 

Since the last update to 1.11.6 the class attribute gets removed and I am left with no (simple) way to detect the source of the span tag.

I tried a quick "workaround" by introducing bogus style information but this gets detected and removed as well. 

Is there a simple way to expand the html-handling to conserve the class attribute and its value? It has been ignored by the rendering engine so far and probably still is.
A more complicated solution might be to remove the unsupported tags but leave attributes of supported tags like span or p unchanged. 

Thanks

**Comment by dpolivaev** [comment]
I am going to treat it as a bug. I have created an issue https://github.com/freeplane/freeplane/issues/1372. Could you please add there an example map where I can see how you used the class attribute? Having examples can help me to reproduce the issue and to find a solution.
